compose/tests/test_docker_compose.py

```diff
@@ -40,6 +39,19 @@ def test_can_build_images_before_spawning_service_via_compose():
     assert "--build" in docker_compose_cmd
 
 
+def test_can_run_specific_services():
+    with patch.object(DockerCompose, "_call_command") as call_mock:
+        with DockerCompose(ROOT, services=["hub", "firefox"]) as compose:
+            ...
+
+    assert compose.services
+    docker_compose_cmd = call_mock.call_args_list[0][1]["cmd"]
+    services_at_the_end = docker_compose_cmd[-2:]
+    assert "firefox" in services_at_the_end
+    assert "hub" in services_at_the_end
+    assert "chrome" not in docker_compose_cmd
```

Comment:
for this one, I replicated the test above and adjusted for my needs. not mega-happy with it, but should do the job for now. I'll be sure to come back and clean this test file up a bit on future contributions.

Reply:
Can we check whether the behavior is as expected? E.g., run only the `firefox` service and check that it works but the `chrome` service does not work. And then the reverse?